jupyter_client/kernelspecapp.py

```diff
@@ -33,6 +33,7 @@ def _kernel_spec_manager_default(self):
 
     def start(self):
         specs = self.kernel_spec_manager.find_kernel_specs()
+        specs = {kname:{"dir":specs[kname],"info":self.kernel_spec_manager.get_kernel_spec(kname).to_dict()} for kname in specs}
```

Comment:
FYI, pep8 we where talking about this morning, that might be relevant: https://www.python.org/dev/peps/pep-0008/